app/javascript/redux/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-param-reassign */
+
```

Comment:
- [ ] [REQUIRED] Please `remove` this comment and try to `fix` the linter issue yourself, it will greatly help you to improve your `experience` 😃